crypto/crypto_test.go

```diff
@@ -1,19 +1,30 @@
 package crypto
 
 import (
+	"regexp"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
 func Test_SHA256(t *testing.T) {
-	t.Run("return SHA256 for string", func(t *testing.T) {
-		SHA256("")
-		t.Error("Missing test case")
+	assert := assert.New(t)
+
+	t.Run("return 64 hexadecimal SHA256 for input string", func(t *testing.T) {
+		value := SHA256("Test String")
+		match, _ := regexp.MatchString("^[A-Fa-f0-9]+$", value)
+		assert.Equal(match, true)
```

Comment:
better you assert the actual value of the SHA256, it's not supposed to change and this way we will be sure it won't.

Reply:
@kengio